### ISSUE TITLE
Clicks are triggering postback on asp.net

### ISSUE THREAD

**Issue by premy** [jestrux/jquery-mini-event-calendar#5]
Clicking on months or days is triggering a postback on asp.net Web Forms. I tried putting e.preventDefault() on the click functions; the postback does not happen any more but then the events are not displayed either.

**Comment by jestrux** [comment]
Hey, could you recreate this problem in a fiddle or codepen so I can review it?

**Comment by premy** [comment]
Hi, thank u for your answer. Thing is I tried using everything as is in an asp.net application. In plain html it works as expected but on an aspx page it causes a postback whenever you click on something on the calendar.
What I tried to do was adding this:
`
**mini_cal.on("click", function(e){
		  e.preventDefault();
		});`**
mini_cal.on("click, focusin", ".a-date", function(){
		    if(!$(this).hasClass('blurred'))
		        showEvent($(this).data('event'));
		});`
But then clicking on a day with event does not show the event in the footer.
